File: 백준/Bronze/10162. 전자레인지/전자레인지.py
import sys
try:
    sys.stdin = open('run/input.txt', 'r')
except FileNotFoundError:
    pass

T = int(input())
A = 300
B = 60
C = 10
a = T // A
T = T % A
b = T // B
T = T % B
c = T // C
T = T % C
if T == 0:
    print(a, b, c)
else:
    print(-1)

File: 백준/Bronze/14470. 전자레인지/전자레인지.ts
const filePath: string =
  process.platform === "linux" ? "/dev/stdin" : "run/input.txt";
const input: string[] = require("fs")
  .readFileSync(filePath)
  .toString()
  .trim()
  .split("\n");

const a: number = +input[0];
const b: number = +input[1];
const c: number = +input[2];
const d: number = +input[3];
const e: number = +input[4];

let result: number = 0;

if (a < 0) {
  result += Math.abs(a) * c;
  result += d;
  result += b * e;
} else {
  result += (b - a) * e;
}

console.log(result);